setup.py

```diff
@@ -73,7 +73,7 @@
 pkg_json = json.loads((HERE / "jupyterlab-server-proxy" / "package.json").read_bytes())
 
 setup_args = dict(
-    name=name,
+    name=name.replace("_", "-"),
```

Comment:
There are other references to the `name`, do you think it is correct for those to not be changes alongside this?

Reply:
I believe the other `name` references should keep the `_` because we're actually referencing a directory in those cases. The entire npm build will break if we rename the original `name` variable (I tried that first, actually).

By only changing it in the `setup_args` I think the entire build process stays the same and the only thing that changes is the name of the `.tar.gz` file.